tests/storage/mpathconf_test.py

```diff
@@ -172,3 +175,52 @@ def test_mpath_empty_conf(fake_mpath_conf):
         revision=mpathconf.REVISION_MISSING,
         private=False,
     )
+
+
+def test_mpathd_parser():
+    mpath_out = """\
+section1 {
+    key1 "value 1"
+    key2 value2
+    key3 3
+}
+section2 {
+    key1 value1
+}
+section3 {
+}
+section4 {
+    entry {
+        key1 "value 1"
+        key2 "value 2"
+        key3 "value 3"
+    }
+    entry {
+        key1 "value1"
+    }
+    entry {
+    }
```

Comment:
We need to test here mixed section with both pairs and other sections (e.g. blacklist).